docs/ref-gitrepo.md

```diff
@@ -136,6 +136,18 @@ spec:
   # must be deleted after deleting the GitRepo.
   # deleteNamespace: false
 
+  # bundles specifies the used-driven bundle definitions
+  # With this way of defining Bundles, Fleet will simply load the specified resources 
+  # along with the options file (if defined), or it will attempt to find a 
+  # fleet.yaml file in the defined base.
```

Comment:
Perhaps we could clarify here that subdirs will not be searched, or that this process is not recursive, unlike in the traditional scanning process.

Reply:
There is a `See [How Repos are scanned](https://fleet.rancher.io/gitrepo-content#how-repos-are-scanned)` line at the end of the block, which should give more context.
I added extra info mentioning about not being recursive in the `how-repos-are-scanned` section.